server/app/views/NorthStarBaseView.java

```diff
@@ -158,6 +176,15 @@ protected ThymeleafModule.PlayThymeleafContext createThymeleafContext(
     return context;
   }
 
+  private String getBaseUrlWithoutScheme() {
+    Optional<String> baseUrlOptional = settingsManifest.getBaseUrl();
+
+    if (baseUrlOptional.isEmpty()) return "";
+
+    String baseUrlWithoutScheme = baseUrlOptional.get();
+    return baseUrlWithoutScheme.replaceFirst("^(https?://)", "");
```

Comment:
(blocking) It's safer to parse this as a `java.net.URL` and get the hostname from that object.